erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -203,6 +203,12 @@ def get_gl_entries(filters, accounting_dimensions):
 		as_dict=1,
 	)
 
+	# fetch party name using frappe.db.get_value
+	for gl_entry in gl_entries:
+    	party_type = gl_entry.party_type
+    	if party_type:
+        	gl_entry.party_name = frappe.db.get_value(party_type, gl_entry.party, f"{party_type.lower()}_name")
```

Comment:
@precihole Can you please add a screenshot of how the report looks post the change? Also, a query is not recommended in for loop, you can get the details in a single query and make a map and use that in the report.

For customers not using naming series for customers and suppliers the same info will be shown twice, so this scenario will also have to be handled